### DISCUSSION TITLE
阿里云是封禁v2ray了吗？

### DISCUSSION THREAD

**Discussion by cekimy** [v2fly/v2ray-core#1463] [Q&A] [answered]
模式：国内阿里云proxy谷歌的v2ray节点，一直运行好好的，前几天没有动这块突然开始上不了了。但是直连GCP的节点没问题，阿里云的haproxy监控看着也是通的，telnet连接阿里云的端口也能连上。
直接在阿里云同样的方式起了一个docker，直接连接也不行，浏览器直接返回
![image](https://user-images.githubusercontent.com/1655088/145557625-b37e6216-47e2-44c2-bb5a-c7ad44bb5aca.png)

**Comment by Kylejustknows** [comment]
封端口了吧?

**Comment by cekimy** [comment, marked as answer]
解决了，应该是伪装的域名没在阿里云备案，现在请求直接被阿里云咔嚓了。多增加一个在阿里云备案过的域名就可以了
